files/galaxy/dynamic_job_rules/pawsey/dynamic_rules/destination_mapper.py

```diff
@@ -15,6 +15,7 @@
     'pulsar_mel3_user@usegalaxy.org.au': 'pulsar-mel3_small',
     'pulsar_paw_user@usegalaxy.org.au': 'pulsar-paw_small',
     'phm1@genome.edu.au': 'pulsar-high-mem1_big',
+    'hm2@genome.edu.au': 'pulsar-high-mem2_big',
```

Comment:
phm2@genome.edu.au is the one with jobs running on high-mem at the moment

Reply:
ahh, I changed it but didn't run it.. I'll revert it